src/Encamina.Enmarcha.SemanticKernel/MemoryStoreExtender.cs

```diff
@@ -126,7 +126,8 @@ public virtual async IAsyncEnumerable<string> BatchUpsertMemoriesAsync(string co
 
         await foreach (var key in asyncKeys)
         {
-            logger.LogInformation(@"Processed memory record {item}.", key);
+            var message = @"Processed memory record {item}.";
```

Comment:
Why do we need this variable? Just make "{Item}" pascal case.

```suggestion
            logger.LogInformation(@"Processed memory record {Item}.", key);
```